omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -78,12 +78,18 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
 
     self.getSelectedDate = self.getSelectedDate || function () {
             var datePicker = angular.element(document.getElementsByName('conditionStartDate'))[0];
-            return datePicker.value;
+            var date = datePicker.value;
+            var formattedDate =date.replace(/-/g,'/');
```

Comment:
Why do you need to first replace `-` with `/`?

Reply:
I noted that when you call the new Date method on a date String with the format of yyyy-MM-dd it considers the time zone when parsing..However when you parse a String with the format yyyy/MM/dd then the timezone isnt considered thus there is no time offset.